content/en/docs/concepts/traffic-management/index.md

```diff
@@ -254,502 +173,243 @@ spec:
-        cookie:
-          regex: "^(.*?;)?(user=jason)(;.*)?$"
-    route:
-    - destination:
-        host: my-svc
-        subset: v2
-  - route:
-    - destination:
-        host: my-svc
-        subset: v1
+route:
+- destination:
+    host: reviews
+    subset: v2
 {{< /text >}}
 
-In the preceding example, there are two routing rules in the `http` section,
-indicated by a leading `-` in front of the first field of each rule.
-
-The first routing rule begins with the `match` field:
-
-- `match` Lists the routing rule's matching conditions.
-
-- `headers` Specifies to look for a match in the header of the request.
-
-- `cookie` Specifies to look for a match in the header's cookie.
-
-- `regex` Specifies the regular expression used to determine a match.
-
-- `route` Specifies where to route the traffic
-   matching the condition. In this case, that traffic is HTTP traffic with the
-   username `jason` in the cookie of the request's header.
-
-- `destination` Specifies the route destination for the traffic matching the rule conditions.
-
-- `host` Specifies the destination's host, `my-svc`.
-
-- `subset` Specifies the destination’s subset for the traffic matching the conditions, `v2` in this case.
-
-The configuration of the second routing rule in the example begins with the
-`route` field with a leading `-`. This rule applies to all traffic that doesn't match the
-conditions specified in the first routing rule.
-
-- `route` Specifies where to route all traffic except for HTTP traffic matching the condition of the previous rule.
-
-- `destination` Specifies the routing rule's destination.
-
-- `host` Specifies the destination's host, `my-svc`.
-
-- `subset`  Specifies the destination’s subset, `v1` in this case.
-
-The following diagram shows the configured traffic routes for the matched traffic and for all other traffic:
-
-{{< image width="40%"
-    link="./virtual-services-6.svg"
-    caption="Configurable traffic route based on the namespace of two application services"
-    >}}
-
-Routing rules are evaluated in a specific order. For details, refer to
-[Precedence](/docs/concepts/traffic-management/#precedence).
-
-#### Match a condition
-
-You can set routing rules that only apply to requests matching a specific
-condition. For example, the following configuration sets up a rule that only
-applies to an incoming request that includes a custom `end-user` header
-containing the exact value `jason`:
+{{< tip >}}
+In this and other examples, we use a Kubernetes short name for the
+destination hosts. When this rule is evaluated, Istio adds a domain suffix based
+on the namespace of the virtual service that contains the routing rule to get
+the fully qualified name for the host. However, this only works if the
+destination hosts and the virtual service are actually in the same Kubernetes
+namespace. Because using the Kubernetes short name can result in
+misconfigurations, we recommend that you specify fully qualified host names in
```

Comment:
meta point: if we recommend it shouldn't we use that in documentation

Reply:
The main reason our samples/examples use short names is so that they are namespace-independent, so users can deploy them in any namespace they'd like. That said, maybe the examples in this concepts section could use long names since, unlike tasks, the user isn't expected to try them. Either that, or maybe we should make this tip a little more clear about why we use short names here (e.g., for simplicity) and in the tasks (namespace independence convenience).

Reply:
I think if we add "...for simplicity" in the tip to clarify it should make it clear that we're just doing it so the examples are very easy to read.

Reply:
Done, PTAL once I've committed everything...

Reply:
That's perfect.

Just a thought: would it be better to make the first part of the tip just an ordinary paragraph, and make the second part a warning instead of a tip? Something like this:
```
In this and the other examples on this page, we use a Kubernetes short name for the
destination hosts for simplicity. When this rule is evaluated, Istio adds a domain suffix based
on the namespace of the virtual service that contains the routing rule to get
the fully qualified name for the host. Using short names in our examples
also means that you can copy and try them in any namespace you like.

{{< warning >}}
Using short names like this only works if the
destination hosts and the virtual service are actually in the same Kubernetes
namespace. Because using the Kubernetes short name can result in
misconfigurations, we recommend that you specify fully qualified host names in
production environments.
{{< /warning >}}
```

Reply:
Oh yeah that works, and means we don't have a giant tip box.